angular_token_auth.js

```diff
@@ -12,6 +12,7 @@
         LOGOUT_REDIRECT_URL: '/logout/',
         AUTH_HEADER_PREFIX: 'Token',
         ALLOWED_HOSTS: [],
+        STORAGE_METHOD: 'auto',
```

Comment:
This string is somewhat redundant as it's not being used, perhaps add it to the `else` statement on L305?